data/json/items/resources/wood.json

```diff
@@ -72,13 +72,42 @@
     "category": "spare_parts",
     "melee_damage": { "bash": 18 }
   },
+  {
+    "type": "GENERIC",
+    "id": "soaking_long_pole",
+    "name": { "str_sp": "soaking long pole" },
+    "description": "A stout, ten-foot pole.  Could be used similarly to a spear. It's currently being soaked with oil and quite slippery. Once the oil penetrates the wood it can be wiped down and used for further projects.",
```

Comment:
```suggestion
    "description": "A stout, ten-foot pole.  Could be used similarly to a spear. It's currently soaked with oil and quite slippery. Once the oil penetrates the wood it can be wiped down and used for further projects.",
```